chapters/en/slides/module6/module6_01.Rmd

```diff
@@ -0,0 +1,340 @@
+    x='deforestation_hectares',
+    y=alt.Y('type', title='', sort='x'))
+```
+
+Notes:
+This slide shows the biggest drivers of deforestation in Brazil
+between the years 2003 and 2013.
+
+We can see that loss of forest 
+is dominated by the demand for beef,
+which leads to permanent clearing of forest
+to create pastures for cattle grazing.
+
+There is also a smaller contributions from small scale clearings,
+fires, and commercial crops.
+
+Although beef is the major culprit in rain forest deforestation,
+it could be the case that our demand for more food has simply increased
+and cattle herds are an effective use of land to create food for human consumption.
+Let's flip to the next slide to find out!
+
+---
+
+## Beef is highly inefficient use of land to generate calories
+
+```{python}
+# TODO save wrangled data
+land_use = pd.read_csv('data/land-use-kcal-poore.csv').rename(columns={'Land use per 1000kcal (Poore & Nemecek, 2018)': 'Land use per 1000kcal (m2)'}).query('Entity in ["Barley", "Maize", "Rice", "Tofu (soybeans)", "Cassava", "Bananas", "Eggs", "Poultry Meat", "Dark Chocolate", "Milk", "Cheese", "Coffee", "Lamb & Mutton", "Beef (beef herd)"]')
+alt.Chart(land_use).mark_bar().encode(
+    alt.X('Land use per 1000kcal (m2)'),
+    alt.Y('Entity', sort='x', title=''))
+```
+
+Notes:
+The plot in this slide reveals that herding cattle for the purpose of beef production
+is one of the most inefficient ways to produce food for human consumption.
+
+Together with the large demand for beef products
+this inefficient use of land creates an unfortunate situation in Brazil
+where a lot of tropical forest is being cut down to make room for new pastures.
+
+When telling a story,
+it is important to not jump to conclusions
+or suggest that solutions to problems are simpler than what they might really be.
+
+So far we have seen that beef is a major contributing factor to deforestation in Brazil,
+but there are many more aspect of the food production chain that we have not considered,
+such as the use of other resources
+
+Changing beef for other products would be highly beneficial,
+but how achievable it is 
+and what cost is associated with making the change
+we don't know from the data we have explored here.
+
+Let's continue our story by looking at the same breakdown for Indonesia,
+is beef production the biggest issue there as well?
+
+---
+
+## Palm oil is the biggest contributor to deforestation in Indonesia
+
+```{python}
+# TODO save wrangled data
+indonesia_drivers = pd.read_csv('data/deforestation-drivers-indonesia.csv').query('2013 > Year > 2003').groupby('Entity').sum().reset_index()
+alt.Chart(indonesia_drivers, title='Palm oil was the leading cause of deforestation between 2003 and 2013 in Indonesia').mark_bar().encode(
+    x='deforestation_hectares',
+    y=alt.Y('Entity', title='', sort='x'))
+```
+
+Notes:
+This slide shows the biggest drivers of deforestation in Indonesia
+between the years 2003 and 2013.
```

Comment:
Again, are we sure we know these are drivers?

Reply:
Yes, see the longer reply above.